hathor/p2p/node_sync.py

```diff
@@ -266,7 +266,7 @@ def stop(self) -> None:
     def is_synced(self) -> bool:
         """ Return True if we are synced.
         """
-        return self.manager.tx_storage.latest_timestamp - self.synced_timestamp <= self.sync_threshold
+        return self.manager.tx_storage.latest_timestamp - self.peer_timestamp <= self.sync_threshold
 
```

Comment:
We should add a comment on the declaration of `self.peer_timestamp` explaining what that is. Also, maybe move its declaration right before `self.synced_timestamp` so it's easy to see that both metrics are available and the difference between them.

Reply:
Done. What do you think?